### DISCUSSION TITLE
Syntax Highlights

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#31025] [Mobile]
### Syntaxs Highlights is seems doesn't fit to what we have on GitHub.

I have noticed that a while ago, but it seems like it's still same. So, I wanted to _report_ that _bad looking_ to compared to Desktop Mode of GitHub. If you also noticed; methods, vars or keywords' color tone not same as Desktop Mode of GitHub either.

#### There is the images

> This is Desktop Mode of GitHub.

![IMG_20220827_130351](https://user-images.githubusercontent.com/71941576/187025453-7b5bf21c-2442-41b9-a0db-a774d41c5575.jpg)

> This is Mobile Version of GitHub.

![IMG_20220827_130041](https://user-images.githubusercontent.com/71941576/187025393-38672efd-c67a-49c8-bb05-ac3c25e3b41f.jpg)

**Comment by candyho** [comment]
Hey, thanks for reporting! Do you mind sharing with us which mobile device are you using, your OS and the app version number? That'd definitely help us to speed up the fix.

**Reply by Neodevils** [reply]
Hello, @candyho!

The device I am using is **Xiaomi, Redmi 8**. AOS is **11**. And GitHub is version is **1.78.0** (Beta).

**Reply by candyho** [reply]
Is this happening on public repository or on private repository as well?

**Reply by Neodevils** [reply]
Of course. It's block of code.

**Reply by candyho** [reply]
Ah I was just wondering if the syntax highlight issue you spotted was from a public or private repository? If it's a public repository, it'd be great if you can share the link to the repo for us to take a look. So far we aren't able to reproduce the issue.

**Reply by Neodevils** [reply]
Sure. Here you can take a [look](https://github.com/NeoAIChan).
